packages/terra-worklist-data-grid/src/subcomponents/Row.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import ThemeContext from 'terra-theme-context';
+import classNames from 'classnames/bind';
+import '../_elementPolyfill';
+import styles from './Row.module.scss';
+import RowSelectionCell from './RowSelectionCell';
+import Cell from './Cell';
+import rowShape from '../proptypes/rowShape';
+import WorklistDataGridPropTypes from '../proptypes/WorklistDataGridPropTypes';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The row's position in the Grid. This is zero based.
+   */
+  rowIndex: PropTypes.number,
+  /**
+   * String that specifies height of the row. Any valid CSS width value is accepted.
+  */
+  height: PropTypes.string,
+  /**
+   * The information about the Row.
+   */
+  row: rowShape.isRequired,
```

Comment:
It is odd that the component `Row` has a prop named `row`. It is as if a `button` component has a `button` prop. We should break out this shape into individual props.

Reply:
@sdadn Let's discuss and we can adjust accordingly.

Reply:
As discussed, we flattened the props